README.md

```diff
@@ -33,6 +33,41 @@ go run main.go
 
 You can access the APIs at http://localhost:8080.
 
+Find APIs at below link:
```

Comment:
- `Find APIs at below link` => `Here is APIs reference used in the website` 
- Add apidoc link in `here` word
- Add the `npm run build`  command to build the project.
- Add how we are running tests in go